tests/integration/test.attachments.js

```diff
@@ -1749,7 +1749,7 @@ adapters.forEach(function (adapter) {
                 cache: true,
                 binary: true
               }, function (err, res) {
-                if (err && err.status === 405) {
+                if (err && err.status === 500) {
```

Comment:
For the record, it seems Firefox 35+ has the blob fix, so it doesn't report any kind of error.

As for previous versions of Firefox, I'm glad we're testing it, but to be fair, our tests won't capture if we get a false negative and revert to base64 when actually blob would be appropriate. But given that base64 works universally, and that Firefox is so good about auto-updating, I'm not really concerned.